#0-add_integer.py

>>> add_integer(1)
99

>>> add_integer(2, 3)
4

>>> add_integer(2.5, 4)
6

>>> add_integer(2, 3.5)
5

>>> add_integer("leslie", 4)
traceback (most recent call last):
TypeError: a must be an integer

>>> add_integer(2, "4")
traceback (most recent call last):
TypeError: b must be an integer

